docs/running_tests.md

```diff
@@ -134,6 +135,23 @@ To focus on a given spec file you can use the environment variable `SPEC_PATH`
 bosh/src$ SPEC_PATH=./spec/integration/cancel_tasks_spec.rb CONCOURSE_TARGET=bosh bundle exec rake fly:integration
 ```
 
+
+## BOSH Release tests
+
+The BOSH Release
+
+### Run the ERB unit tests
+
+Install the Gem dependencies.
+```
+(cd src && bundle install)
+```
+
+Run the ERB unit tests.
+```
+scripts/test-unit-erb
```

Comment:
This should be `../scripts/test-unit-erb` because we end up in the `bosh/src` folder after executing the previous command `cd src && bundle install` or specify that it needs to be executed from the `bosh` folder.

Reply:
Well that wasn't the idea, because `(cd src && bundle install)` changes the current working directory for the sub-shell only.